packages/core/src/switch/Switch.css

```diff
@@ -102,7 +102,8 @@
 }
 
 @media (prefers-reduced-motion) {
-  .saltSwitch-thumb {
+  .saltSwitch-input:focus-visible + .saltSwitch-track .saltSwitch-thumb,
+  .saltSwitch:hover .saltSwitch-thumb {
```

Comment:
Why the component changes?

Reply:
with the bundled css and a switch that is checked by default, you see the transition of the switch thumb from left to right.  The change ensures that the transition only happens if the switch input is focused